File: Assets/[2DSANDBOX]/Scripts/MenuScripts/StartGame.cs
using UnityEngine;
using UnityEngine.SceneManagement;

public class StartGame : MonoBehaviour {

	public Texture2D playNormal;
	public Texture2D playHover;

	void OnMouseEnter(){
		GetComponent<GUITexture>().texture = playHover;
	}

	void OnMouseExit(){
		GetComponent<GUITexture>().texture = playNormal;
	}

	void OnMouseDown()
	{
		SceneManager.LoadScene("SelectPlayer");
	}

	// Use this for initialization
	void Start ()
    {
        Application.targetFrameRate = 60;
	}
}

File: Assets/[2DSANDBOX]/Scripts/MenuScripts/StartGame.js
//@McGameStudios 2014-2015 
//mcgamestudios.za.pl /

#pragma strict
var playNormal : Texture2D;
var playHover : Texture2D;



function Start () {

}

function OnMouseEnter () {

	GetComponent.<GUITexture>().texture = playHover;

}

function OnMouseExit () {

	GetComponent.<GUITexture>().texture = playNormal;

}

function OnMouseDown ()
 {
 	//Nilupul
 	//SaveGame.Instance.IsSaveGame = false;
	Application.LoadLevel('World');

}